GiniVision/Classes/GiniVisionUtils.swift

```diff
@@ -159,4 +159,25 @@ internal struct Colors {
             alpha: CGFloat(1.0)
         )
     }
-}
\ No newline at end of file
+}
+
+/**
+    Set the status bar style when ViewControllerBasedStatusBarAppearance is disabled.
+ */
+
+internal func setStatusBarStyleIfNeeded(to statusBarStyle: UIStatusBarStyle,
```

Comment:
I kind of hate the name `IfNeeded` in a method. Sometimes it's necessary, but I'd consider removing it here. Maybe just have the comment in a doc snippet above.

Reply:
Given the previous comment, it does not make sense anymore to have the `IfNeeded`👍